pkg/parse/parse.go

```diff
@@ -414,6 +428,130 @@ func GetBindMount(args []string, contextDir string) (specs.Mount, error) {
 	return newMount, nil
 }
 
+// GetCacheMount parses a single cache mount entry from the --mount flag.
+func GetCacheMount(args []string) (specs.Mount, error) {
+	var err error
+	newMount := specs.Mount{
+		Type: TypeBind,
+	}
+
+	setDest := false
```

Comment:
I think this section would look better if it was in a 
var (
   setDest bool
  setShared bool
  setReadonly bool
...
) 
block.

Reply:
Thanks i'll fix this.